### PR TITLE
feat: upgrade Kroma mainnet to v1.4.2

### PR DESCRIPTION
**Pull request by 0xHansLee**
Change Ecotone activation time for Kroma mainnet.
The activation time is `Thu Apr 25 2024 08:00:01` (unix timestamp `1714032001`).

### CODE REVIEW

File: .env.mainnet [hunk 1]
```diff
@@ -10,9 +10,11 @@ IMAGE_TAG__KROMA_PROVER=v0.1.4
 # L1 RPC endpoint.
 # You can check and use one of the list of RPC endpoints: https://chainlist.org/chain/1
 # Or you can also use Alchemy or Infura, etc.
+# After Ecotone activation, you need also L1 Beacon endpoint to get blob data.
```
Comment (seolaoh) [inline .env.mainnet:13]: Please change the image version of geth, node, validator.

File: upgrades/v1.4.2.md [hunk 2]
```diff
@@ -1,17 +1,27 @@
...
 
-- Kroma sepolia: `Wed Apr 17 2024 08:00:00 UTC`
+- Kroma Sepolia: `Wed Apr 17 2024 08:00:00 UTC`
+- Kroma Mainnet: `Thu Apr 25 2024 08:00:01 UTC`
 
 ## Guides for upgrade
 
+### Pull the latest version of kroma-up
+
+First of all, pull the latest kroma-up and checkout to v1.4.1.
```
Comment (seolaoh) [inline upgrades/v1.4.2.md]:
```suggestion
First of all, pull the latest kroma-up and checkout to v1.4.2.
```

File: upgrades/v1.4.2.md [hunk 3]
```diff
@@ -1,17 +1,27 @@
...
+
+```bash
+git pull origin main
+git checkout tags/v1.4.1
```
Comment (seolaoh) [inline upgrades/v1.4.2.md]:
```suggestion
git checkout tags/v1.4.2
```

**Review by seolaoh — APPROVED** [review]
LGTM

**Review by kangsorang — APPROVED** [review]

**Review by sm-stack — APPROVED** [review]

**Pull request merged**